lib/cmd/common/initlocal.js

```diff
@@ -125,64 +128,82 @@ function modifyTagsToLocal(appContainer, tagsToModify, localDirectory, cb) {
 function constructContainerFromIndex(indexFile, cb) {
   var outputFile;
   var appContainer = cheerio.load(indexFile);
-  setBodyToPlaceholder(appContainer, function (err, appContainer) {
-    if (err) return cb(err);
+  console.log(appContainer.html());
+  setBodyToPlaceholder(appContainer, function(err, appContainer) {
+    if (err) {
+      return cb(err);
+    }
     var tagsToModify = [
       {tag: "script", attr: "src"},
       {tag: "link", attr: "href"}
     ];
 
-    modifyTagsToLocal(appContainer, tagsToModify, localDirectory, function (err, appContainer, fileList) {
-      if (err) return cb(err);
+    modifyTagsToLocal(appContainer, tagsToModify, localDirectory, function(err, appContainer, fileList) {
+      if (err) {
+        return cb(err);
+      }
       var hostRegExp = new RegExp('"host"\\s*:\\s*"' + fhreq.getFeedHenryUrl() + '?"');
       outputFile = appContainer.html().replace(hostRegExp, '"host":"' + localHostUrl() + '"');
+      console.log(hostRegExp);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)